### PR TITLE
docs: setup instructions for tari swarm

### PR DESCRIPTION
**Pull request by MCozhusheck**
Description
---
Update README instruction how to setup `tari_swarm_daemon`.

Breaking Changes
---

- [x] None
- [ ] Requires data directory to be deleted
- [ ] Other - Please specify

### CODE REVIEW

File: applications/tari_swarm_daemon/docker/README.md [hunk 1]
```diff
@@ -8,8 +8,9 @@ cd sources
 git clone https://github.com/tari-project/tari.git
 git clone https://github.com/tari-project/tari-dan.git
 git clone https://github.com/tari-project/tari-connector.git
-cp -v applications/tari_swarm/docker/cross-compile-aarch64.sh .
-docker build -f applications/tari_swarm/docker/tari_swarm.Dockerfile \
+cp -v tari-dan/applications/tari_swarm_daemon/docker/cross-compile-aarch64.sh .
+cd ..
+docker build -f sources/tari-dan/applications/tari_swarm_daemon/docker/tari_swarm.Dockerfile \
```
Comment (sdbondi) [inline applications/tari_swarm_daemon/docker/README.md:13]: Would this work too? Haven't tested but I think an escape newline is missing 

```suggestion
docker build -f tari-dan/applications/tari_swarm_daemon/docker/tari_swarm.Dockerfile \ 
```

Comment (MCozhusheck) [reply applications/tari_swarm_daemon/docker/README.md:13]: building inside `sources` dir won't work as `tari_swarm.Dockerfile` has at [line 151](https://github.com/tari-project/tari-dan/blob/development/applications/tari_swarm_daemon/docker/tari_swarm.Dockerfile#L151)
```dockerfile
ADD sources/tari-dan /home/tari/sources/tari-dan
```
So it's assumes we are building in a parent directory of `sources` dir

Comment (MCozhusheck) [reply applications/tari_swarm_daemon/docker/README.md:13]: Another way would be to change Dockerfile but it would be more work than simple `cd ..`

Comment (sdbondi) [reply applications/tari_swarm_daemon/docker/README.md:13]: Ah thanks, I was reviewing on my phone so didn't see the whole context

**Review by sdbondi — APPROVED** [review]

**Pull request merged**